src/en/anchira/src/eu/kanade/tachiyomi/extension/en/anchira/AnchiraHelper.kt

```diff
@@ -25,4 +29,31 @@ object AnchiraHelper {
             }
         }
         .joinToString(", ") { it }
+
+    fun createChapter(entry: Entry, response: Response, anchiraData: List<EntryKey>) =
+        SChapter.create().apply {
+            val ch =
+                Regex("(?<!20\\d\\d-)\\b[\\d.]{1,4}$").find(entry.title)?.value?.trim('.') ?: "1"
```

Comment:
make this regex a constant variable in this file and use that in main class as well

Reply:
I meant making the regex itself a constant

```
val CHAPTER_SUFFIX_REGEX = Regex("(?<!20\\d\\d-)\\b[\\d.]{1,4}$")
```

Reply:
In my defense, you did say _constant_ and those only take primitives xD